tests/solids/diamondC_2x1x1_pp/CMakeLists.txt

```diff
@@ -60,6 +60,15 @@ ENDIF()
                     1 DIAMOND2_DMC_SCALARS # DMC
                     )
 
+  QMC_RUN_AND_CHECK(short-diamondC_2x1x1_pp-dmcbatch_sdj
+                    "${CMAKE_SOURCE_DIR}/tests/solids/diamondC_2x1x1_pp"
+                    qmc_short
+                    qmc_short_dmcbatch.in.xml
```

Comment:
This test uses the reference values of qmc_short_vmc_dmc.in.xml but input parameters are very different. Need a fix later.

Reply:
The test is definitely invalid and I expect it to fail.  But it doesn't crash and produces scalar.dat output.  Population control is suppressed by forcing the no_branch call on WalkerControl.  Either before or after this PR is merged that is the next task for batched DMC.